src/Support/UserAgentParser.php

```diff
@@ -18,6 +18,7 @@ class UserAgentParser
 
     public function __construct($basePath, $userAgent = null)
     {
+        $userAgent = $_SERVER['HTTP_USER_AGENT'];
```

Comment:
If you set it in here, there's no point in having the parameter on the construct, it will be always overwritten.

Reply:
O yeah, i forgot, thanks for your reply.

Reply:
I Just Push code to resolve thats issues. Thank You

Reply:
I'm facing the same issue how can I solve it ??